src/login/pam_systemd.c

```diff
@@ -404,7 +415,15 @@ _public_ PAM_EXTERN int pam_sm_open_session(
                                remote,
                                remote_user,
                                remote_host,
-                               0);
+                               7, "MemoryMax", "s", memory_max,
```

Comment:
these types aren't right. this must be passed as uint64_t, i.e. "t", rather than "s". You need to parse the props before passing them. Similar for the other props.

Reply:
Also, I am pretty sure we should send the minimal set of props here, i.e. if some prop isn't configured, don't send it. That means you can't use the sd_bus_call_method() convenience helper anymore though, you need to put together the message manually instead and then send it.

Reply:
The types weren't right, that's exactly why it didn't work...  Thanks for the review!